src/plugins_3rdparty/primer3/src/Primer3Query.cpp

```diff
@@ -101,27 +102,27 @@ Task* QDPrimerActor::getAlgorithmTask(const QVector<U2Region>& /*location*/) {
     int numRet = cfg->getParameter(NUM_RETURN_ATTR)->getAttributeValueWithoutScript<int>();
     settings->setIntProperty("PRIMER_NUM_RETURN", numRet);
 
-    /*qreal maxMispriming = cfg->getParameter(MAX_MISPRIMING_ATTR)->getAttributeValueWithoutScript<double>();
-    settings->setAlignProperty("PRIMER_MAX_MISPRIMING", maxMispriming);
-    assert(settings->getAlignPropertyList().contains("PRIMER_MAX_MISPRIMING"));*/
+    qreal maxMispriming = cfg->getParameter(MAX_MISPRIMING_ATTR)->getAttributeValueWithoutScript<double>();
+    settings->setDoubleProperty("PRIMER_MAX_LIBRARY_MISPRIMING", maxMispriming);
+    assert(settings->getDoublePropertyList().contains("PRIMER_MAX_LIBRARY_MISPRIMING"));
```

Comment:
Do not use asserts, use 'SAFE_POINTS' so we detect the problem in release tests

Reply:
And I've never used asserts :)

Reply:
This remark applies to the entire UGENE and, in my opinion, should be resolved in a separate commit.

Reply:
✓

Reply:
I usually add comments like these about changed lines: once the line is changed it may be fixed too.
Not a blocker anyway.